test/controllers/update/PlatformOperatorControllerSpec.scala

```diff
@@ -38,58 +37,44 @@ import scala.concurrent.Future
 
 class PlatformOperatorControllerSpec extends SpecBase with MockitoSugar with BeforeAndAfterEach {
 
-  private val mockConnector = mock[PlatformOperatorConnector]
+  private val mockPlatformOperatorConnector = mock[PlatformOperatorConnector]
+  private val mockSubmissionsConnector = mock[SubmissionsConnector]
   private val mockRepository = mock[SessionRepository]
 
   override def beforeEach(): Unit = {
-    Mockito.reset(mockConnector, mockRepository)
+    Mockito.reset(mockPlatformOperatorConnector, mockSubmissionsConnector, mockRepository)
     super.beforeEach()
   }
 
   "Platform Operator controller" - {
-
     "must get platform operator details, save them, and show the correct view" in {
-
-      val operator = PlatformOperator(
-        operatorId = "operatorId",
-        operatorName = "operatorName",
-        tinDetails = Seq.empty,
-        businessName = None,
-        tradingName = None,
-        primaryContactDetails = ContactDetails(None, "primaryContactName", "primaryEmail"),
-        secondaryContactDetails = None,
-        addressDetails = AddressDetails("line1", None, None, None, Some("postCode"), None),
-        notifications = Seq.empty
-      )
-
-      when(mockConnector.viewPlatformOperator(any())(any())) thenReturn Future.successful(operator)
+      when(mockPlatformOperatorConnector.viewPlatformOperator(any())(any())) thenReturn Future.successful(aPlatformOperator)
+      when(mockSubmissionsConnector.submissionsExist(any)(any)) thenReturn Future.successful(true)
```

Comment:
Need a test for when submissionsExist returns future.failure
Need a test for when assumedReportsExist returns future.failure